frontend/src/components/admin/question-creation/question-elements/modals/fraction/EditFractionModal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from "react";
+import { useDisclosure } from "@chakra-ui/react";
+
+import QuestionEditorContext from "../../../../../../contexts/QuestionEditorContext";
+import type { FractionMetadata } from "../../../../../../types/QuestionMetadataTypes";
+import { updatedQuestionElement } from "../../../../../../utils/QuestionUtils";
+
+import FractionModal from "./FractionModal";
+
+interface EditFractionModalProps {
+  id: string;
+  data: FractionMetadata;
+  isOpen: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const EditFractionModal = ({
+  id,
+  data,
+  isOpen,
+  setOpen,
+}: EditFractionModalProps): React.ReactElement => {
+  const { onClose } = useDisclosure();
```

Comment:
Similar comment about `useDisclosure` from before. In this case though I think it might be better to use a `useDisclosure` in the parent component (admin `FractionElement`) and pass down the `isOpen` and `onClose` props.